HelloWorld/HelloWorld.cs

```diff
@@ -1,12 +1,25 @@
 ﻿using System;
 
-namespace HelloWorld
+public class  Program
 {
-    class Program
+    public static void Main()
     {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
-    }
+        string name = "";
+        int age = 0;
+        int year = 0;
+          
+           
+        Console.WriteLine("Please enter your name: ");
+        name = Console.ReadLine();
+        Console.WriteLine("Please enter your age: ");
+        age = Convert.ToInt32(Console.ReadLine());
+        Console.WriteLine("Please enter the year: ");
```

Comment:
Put on your devil's advocate hat and think about how this question might cause problems. If you want to print out the year someone was born. What happens when the user (the person interacting with your application) is has a birthday after the day they are using your application? What about before?